### ISSUE TITLE
can't create docker service

### ISSUE THREAD

**Issue by Romanicus** [LongLiveCHIEF/wsl2-docker-for-win#3]
allways when i run `sudo service docker enable` I get the message `docker: unrecognized service`. I'll execute all other commands as shown.. Do you have any suggetions what the error could be?

**Comment by Romanicus** [comment]
ok, I found it. I didn't 'install' Docker. This is missing in your manual. So if you add the one-liner for installation `apt-get install docker-ce docker-ce-cli containerd.io` it would be less confusing. You mentioned at the beginning of your manual that it is nearly the same as in the official documentation, but I didn't remember that when I followed your instructions.

**Comment by LongLiveCHIEF** [comment]
You are totally right! How could I have a [Install docker](https://github.com/LongLiveCHIEF/wsl2-docker-for-win#install-docker-on-ubuntu) step that does everything _but_ install docker!?

Good catch, will fix as soon as I can spare a few minutes :smiley:
